microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -29,6 +30,12 @@ When hosting the WebView2 control, your app has access to the following features
 | [Performance and debugging](#performance-and-debugging) | Analyze and debug performance, handle performance-related events, and manage memory usage to increase the responsiveness of your app. |
 | [Chrome Developer Protocol (CDP)](#chrome-developer-protocol-cdp) | Instrument, inspect, debug, and profile Chromium-based browsers.  The Chrome DevTools Protocol is the foundation for the Microsoft Edge DevTools.  Use the Chrome DevTools Protocol for features that aren't implemented in the WebView2 platform. |
 
+<!-- maintenance notes: add table rows for any new h2 sections -->
+
+<!-- todo: if "Managing SmartScreen" is an h2 section, add a row for it above
```

Comment:
@novac42 - if "Managing SmartScreen" ends up as an h2 section, add a row for it above.

Reply:
removed v112 items from this PR